Make congestion control more self-contained (#3463)
quicwg/base-drafts: draft-ietf-quic-recovery.md

# Congestion Control {#congestion-control}

> This document specifies a congestion controller for QUIC similar to

Reviewer: ```suggestion
This document specifies a congestion controller for QUIC based on
```

congestion. This document specifies an endpoint's response when its peer
receives packets with the Congestion Experienced codepoint.

## Initial and Minimum Congestion Window {#initial-cwnd}

QUIC begins every connection in slow start with the congestion window set to
an initial value.  The RECOMMENDED initial value is the minimum of

> 10 * max_datagram_size and max(2 * max_datagram_size, 14720)).  This follows

Reviewer 2: Endpoints SHOULD use an initial congestion window of 10 times the maximum datagram size (max_datagram_size), limited to the larger of 14720 or twice the maximum datagram size.

## Slow Start

QUIC begins every connection in slow start with the congestion window set to
an initial value.  The RECOMMENDED initial value is the minimum of
10 * max_datagram_size and max(2 * max_datagram_size, 14720)).  This follows

> the analysis and recommendations in {{?RFC6928}}, increasing the byte limit to

Reviewer 3: The justification is repeated in Appendix B.1. I like it better here, but maybe make B.1 more concise and just point to here?

Author: SG, I made B.1 more consise and gave this and the min CWND their own sub-section.

kInitialWindow:
: Default limit on the initial amount of data in flight, in bytes.

>   The RECOMMENDED value is the minimum of 10 * max_datagram_size and

Reviewer 2: use min(10 * max_datagram_size, max(2 * max_datagram_size, 14720)))
